open-craft/labxchange-xblocks
# When rendering public_view of composite xblocks call public_view of child xblocks

**Testing Instructions**
* Pull this branch. The repo is in `devstack/src/`.
* Do `make lms-restart`.
* As a logged in user, view all the xblocks types from the LX frontend and check that they render correctly.
* As a logged out user, view all the xblocks types from the LX frontend and check that they render correctly.
